Improved readability (#1735)
enkidevs/curriculum: react/introduction/intro-react/react-what-i-need-to-use-react.md

---
## Content

> Being a truly flexible library, you can not only create applications from scratch with **React**, but also gradually include them in to your codebase.

Reviewer: Technically, you would gradually introduce React to the codebase, not applications.

Maybe say?

```suggestion
Being a truly flexible library, you can not only create applications from scratch with **React** but also gradually include it into your codebase.
```
